pkgs/by-name/gp/gpustat/package.nix

```
@@ -0,0 +1,62 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, fontconfig
+, libGL
+, xorg
+, cmake
+, libxkbcommon
+, wayland
+, makeWrapper
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "gpustat";
+  version = "0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "arduano";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-zJuadQfyGJemjvkigC77Ql112w5alFrFg3hRnX9io58=";
+  };
+
+  cargoSha256 = "sha256-GNYEc7XYmO1tSqJx0KVPBW/Wq8PmjDpJBle0NYj7bkI=";
+
+  nativeBuildInputs = [ pkg-config cmake makeWrapper ];
+  buildInputs = [
+    fontconfig
+    libGL
+    libxkbcommon
+    xorg.libX11
+    xorg.libXcursor
+    xorg.libXi
+    xorg.libXrandr
+    wayland
+  ];
+
+  postInstall = ''
+    mkdir -p $out/share/applications
+    mkdir -p $out/share/pixmaps
+
+    cp $src/assets/gpustat.desktop $out/share/applications
+    cp $src/assets/icon_* $out/share/pixmaps
+  '';
+
+  # Wrap the program in a script that sets the LD_LIBRARY_PATH environment variable
+  # so that it can find the shared libraries it depends on.
+  # https://github.com/emilk/egui/issues/2486
+  postFixup = ''
+    wrapProgram $out/bin/gpustat --prefix LD_LIBRARY_PATH : "${lib.makeLibraryPath buildInputs}:/run/opengl-driver/lib"
+  '';
+
+  meta = with lib; {
+    description = "A simple utility for viewing GPU utilization.";
+    homepage = "https://github.com/arduano/gpustat";
+    license = with licenses; [ asl20 ];
```

```suggestion
    license = licenses.asl20;
```
bring in line with common style for single-license packages.